plugin/collectd-setup

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# collectd-setup Startup script to setup collectd-mlab switch configuration.
+# chkconfig: - 85 16
+# description: collectd-setup performs auto-discovery of the local switch
+#   configuration to complete collectd setup. collectd-setup should run before
+#   collectd.
+
+
+# NOTE: Perform SNMP auto-discovery for switch configuration.
+# If we cannot update the snmp community string, then the script should halt.
+COLLECTD_SNMP="/etc/collectd-snmp.conf"
+COMMUNITY="/home/mlab_utility/config/snmp.community"
```

Comment:
I'll use `conf`. I need to update the slice data backup script to use the `conf` path instead.